backend/kesaseteli/kesaseteli/settings.py

```diff
@@ -155,11 +157,59 @@
 CSRF_COOKIE_DOMAIN = env.str("CSRF_COOKIE_DOMAIN")
 CSRF_TRUSTED_ORIGINS = env.list("CSRF_TRUSTED_ORIGINS")
 
+# Audit logging
+AUDIT_LOG_ORIGIN = env.str("AUDIT_LOG_ORIGIN")
+AUDIT_LOG_FILENAME = env.str("AUDIT_LOG_FILENAME")
```

Comment:
If running in a kubernetes pod, we can't trust that any local file is really persisted -> need to develop some solution (with IBM?) for storing the audit log. Also, if running multiple pods in prod, need to merge the logs? But that's outside of the scope of this ticket I guess?

Reply:
This is how helsinki profile has implemented it at the moment so let's go with this for now.

The audit log storing is still an open issue and might well change, we will know more next week / after the holidays about storing the logs and how we transfer them to elastic cloud. One option is that we will store the logs in database which would then affect this solution as well.